build/static-files.js

```diff
@@ -32,7 +32,7 @@ export const copyPatterns = [
     },
     { from: 'img', to: 'img' },
     {
-        from: 'node_modules/webextension-polyfill/dist/browser-polyfill.js',
+        from: 'node_modules/webextension-polyfill-ts/dist/',
```

Comment:
Why is this needed? 
I still see `lib/browser-polyfill.js` in the `extension/` dir post-build, although I would expect this change to break that (`node_modules/webextension-polyfill-ts/dist/` package doesn't seem to contain `browser-polyfill.js`).

Reply:
This was just for testing.